src/main/java/Main.java

```diff
@@ -0,0 +1,19 @@
+import dao.AccountDao;
+import dao.DBConfiguration;
+
+public class Main {
+
+    public static void main(String[] args) {
+        if (args[0].equals("-fileName") && args.length == 2) {
```

Comment:
If args is empty, the `ArrayIndexOutOfBoundsException` still will be thrown. You can put the length check in the first place.

Reply:
Fixed